specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -451,10 +651,32 @@
             "$ref": "#/definitions/PrivateLinkServiceConnection"
           },
           "description": "A grouping of information about the connection to the remote resource. Used when the network admin does not have access to approve connections to the remote resource."
+        },
+        "customDnsConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomDnsConfig"
+          },
+          "description": "A grouping of information about dns configurations connecting to the remote resource."
         }
       },
       "description": "Properties of the private endpoint."
     },
+    "CustomDnsConfig":{
+      "properties": {
+        "fqdn": {
+          "type": "string",
+          "description": "The fqdn for the remote resource."
+        },
+        "ipAddresses": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
should it be the private ip of private endpoint?

Reply:
Following up with manager, after review of the doc you sent, I believe so since this is how you'll connect to your DNS resource. (Same as in first thing you asked haha)